tests/test_snmpserver.rb

```diff
@@ -145,23 +176,34 @@ def test_snmpserver_sys_contact_set_special_chars
   def test_snmpserver_sys_contact_set_default
     snmpserver = SnmpServer.new
     snmpserver.contact = snmpserver.default_contact
-    line = assert_show_match(command: 'show snmp | no-more',
-                             pattern: /sys contact: [^\n\r]*/)
-    contact = ''
-    contact = line.to_s.gsub('sys contact:', '').strip unless line.nil?
+    if platform == :nexus
+      line = assert_show_match(command: 'show snmp | no-more',
+                               pattern: /sys contact: [^\n\r]*/)
+      contact = ''
+      contact = line.to_s.gsub('sys contact:', '').strip unless line.nil?
+    else
+      refute_show_match(command: 'show running-config snmp-server',
+                        pattern: /snmp-server contact [^\n\r]+/)
+      contact = ''
```

Comment:
@chburket the iOS_xr does not display contact if it is not set. Therefore there is nothing to gsub.